runtime/mangata-kusama/src/lib.rs

```diff
@@ -1670,6 +1699,9 @@ construct_runtime!(
 		} = 1,
 		Timestamp: pallet_timestamp::{Pallet, Call, Storage, Inherent} = 2,
 		ParachainInfo: parachain_info::{Pallet, Storage, Config} = 3,
+		Utility: pallet_utility_mangata::{Pallet, Call, Event} = 4,
+		Proxy: pallet_proxy::{Pallet, Call, Storage, Event<T>} = 5,
+		Maintenance: pallet_maintenance::{Pallet, Call, Storage, Event<T>} = 6,
 
 		// Monetary stuff.
```

Comment:
xyk needs to stay unchanged due to OAK's XCM transacts for autocompound

monetary ids = 10->29